doc/source/whatsnew/v0.21.0.txt

```diff
@@ -138,6 +138,16 @@ Backwards incompatible API changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 
+.. _whatsnew_0210.api_breaking.Period:
+
+pd.Period objects are now immutable
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:class:`Period` will now raise an ``AttributeError`` when a user tries to
+assign a new value to the `ordinal` or `freq` attributes (:issue:`17116`).
+=======
```

Comment:
OK.  RST is another area where I'm figuring it out as I go.  About to push a change that _I think_ corresponds to what you're saying.